buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/TestClustersPlugin.java

```diff
@@ -95,18 +91,14 @@ public void apply(Project project) {
                     "ES distributions and plugins."
             );
 
-            // When running in the Daemon it's possible for this to hold references to past
-            usedClusters.clear();
-            claimsInventory.clear();
-            runningClusters.clear();
-
             // We have a single task to sync the helper configuration to "artifacts dir"
             // the clusters will look for artifacts there based on the naming conventions.
             // Tasks that use a cluster will add this as a dependency automatically so it's guaranteed to run early in
             // the build.
             rootProject.getTasks().create(SYNC_ARTIFACTS_TASK_NAME, sync -> {
                 sync.getInputs().files((Callable<FileCollection>) helperConfiguration::getAsFileTree);
-                sync.getOutputs().dir(getTestClustersConfigurationExtractDir(project));
+                sync.getOutputs().dir(new File(project.getRootProject().getBuildDir(), "testclusters/extract"));
+                // NOTE: Gradle doesn't allow a lambda here ( fails at runtime )
```

Comment:
Really? Do you have an example of the error handy?

Reply:
I haven't saved it, but it's  saying exactly that, that passing a lambda here doesn't work. Probably has to do with some internal limitation. I added the comment because already bit me twice as IDEA suggests it could be a lambda.

Reply:
Interesting. SAM-coercion should kick in here.